cmd/kube-controller-manager/app/patch_gc.go

```diff
@@ -13,7 +13,6 @@ func applyOpenShiftGCConfig(controllerManager *config.Config) error {
 		// explicitly disabled from GC for now - not enough value to track them
 		gcconfig.GroupResource{Group: "authorization.openshift.io", Resource: "rolebindingrestrictions"},
 		gcconfig.GroupResource{Group: "network.openshift.io", Resource: "clusternetworks"},
-		gcconfig.GroupResource{Group: "network.openshift.io", Resource: "egressnetworkpolicies"},
 		gcconfig.GroupResource{Group: "network.openshift.io", Resource: "hostsubnets"},
 		gcconfig.GroupResource{Group: "network.openshift.io", Resource: "netnamespaces"},
```

Comment:
I'm not familiar with this code, but EgressNetworkPolicies is definitely different from the other `network.openshift.io` types, in that it is managed entirely by the user, not by openshift-sdn. So I'm assuming this is correct.